unleashandroidsdk/build.gradle.kts

```diff
@@ -173,7 +173,8 @@ val jacocoTestReport by tasks.register<JacocoReport>("jacocoTestReport") {
     }
 
     val fileTreeConfig: (ConfigurableFileTree) -> Unit = {
-        it.exclude("**/R.class", "**/R$*.class", "**/BuildConfig.*", "**/Manifest*.*", "android/**/*.*")
+        it.exclude("**/R.class", "**/R$*.class", "**/BuildConfig.*", "**/Manifest*.*", "android/**/*.*",
+            "**/data/**", "**/errors/**", "**/events/**")
```

Comment:
Ignoring some data classes (maybe I can be more specific and refer to the specific data classes)